test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -425,14 +364,56 @@ public void close()
         forceKill();
     }
 
+    private static final class StreamGobbler implements Runnable
+    {
+        private final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        private final InputStream input;
+        private final CompletableFuture<ByteArrayOutputStream> whenComplete;
+
+        private StreamGobbler(InputStream input, CompletableFuture<ByteArrayOutputStream> whenComplete)
+        {
+            this.input = input;
+            this.whenComplete = whenComplete;
+        }
+
+        public void run()
+        {
+            byte[] buffer = new byte[1 << 13];
```

Comment:
Replace `1 << 13` imo for a human readable equivalent of 8K as it's a constant and you're not in a hot path with a variable either.

Reply:
`1 << 13` is a compile time constant, so javac compiles it down to 8k

```
$ javap -v -p -cp build/classes/main/:build/test/classes/ 'org.apache.cassandra.tools.ToolRunner$StreamGobbler'
...
public void run();
    descriptor: ()V
    flags: ACC_PUBLIC
    Code:
      stack=4, locals=3, args_size=1
         0: sipush        8192
         3: newarray       byte
```

I will change to a constant.